src/controller/CHIPDeviceController.h

```diff
@@ -706,6 +706,25 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
      */
     CHIP_ERROR ICDRegistrationInfoReady();
 
+    /**
+     * @brief
+     *   This function is called by the upper layer application to indicate that the required terms and conditions
+     *   acknowledgements have been set. This function should be called after the terms and conditions bitmask and version
```

Comment:
"Have been set" where and how?

Reply:
After discussing offline, I have removed the wait-state.